src/AMSlib/wf/resource_manager.hpp

```diff
@@ -116,37 +109,26 @@ class ResourceManager
     RMAllocators[dev]->deallocate(data);
   }
 
-  /** @brief registers an external pointer in the umpire allocation records.
-   *  @param[in] ptr pointer to memory to register.
-   *  @param[in] nBytes number of bytes to register.
-   *  @param[in] dev resource to register the memory to.
-   *  @return void.
-   */
-  PERFFASPECT()
-  void registerExternal(void* ptr, size_t nBytes, AMSResourceType dev)
-  {
-    RMAllocators[dev]->registerPtr(ptr, nBytes);
-  }
-
-  /** @brief removes a registered external pointer from the umpire allocation records.
-   *  @param[in] ptr pointer to memory to de-register.
-   *  @return void.
-   */
-  void deregisterExternal(void* ptr) { AMSAllocator::deregisterPtr(ptr); }
-
   /** @brief copy values from src to destination regardless of their memory location.
    *  @tparam TypeInValue type of pointers
    *  @param[in] src Source memory pointer.
    *  @param[out] dest destination memory pointer.
-   *  @param[in] size number of bytes to copy. (When 0 copies entire allocated area)
+   *  @param[in] size number of values to copy.
```

Comment:
does it always assume the value type does not contain nested pointers? add it to the comment?

Reply:
Resolved.